lua/configs/which-key-register.lua

```diff
@@ -36,6 +37,8 @@ if is_available "telescope.nvim" then
   init_table("n", "<leader>", "g")
 end
 
+if is_available "nvim-dap" then init_table("n", "<leader>", "D") end
```

Comment:
capitals are annoying. anyway this could be change? I know the dashboard mapping would be backwards incompatible, possibly

Reply:
I agree, but in this case I think it will be important to not impose breaking changes. This also aligns with how VS Code configures debugging bindings which is a nice unification for people migrating back and forth. The main thing here is also for most used actions such as toggling break points/starting/stopping are all accessible through function keys which are much more convenient but take learning. For this reason we have added hints to the function key equivalent to relevant mappings' descriptions.

Outside of this I'm not sure if there is a good mapping other than `D` that would be applicable here. If you have a recommendation I would love to hear! (without breaking change of mapping `d`. We could also push this to AstroNvim v3 which I'm totally fine with and do the breaking change of `<leader>d`